cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -383,7 +395,7 @@ data:
       num_threads 2
     </match>
 metadata:
-  name: fluentd-gcp-config-v1.1
+  name: fluentd-gcp-config-v1.1.1
```

Comment:
why not 1.2? the change is pretty significant and I believe just major and minor is enough for config purposes.

Reply:
Done, thanks. Kept patch version to be consistent with the semver